internal/query/custom_text.go

```diff
@@ -915,6 +918,60 @@ func registrationOptionKeyToDomain(text *CustomText, result *domain.CustomLoginT
 	}
 }
 
+func externalRegistrationUserKeyToDomain(text *CustomText, result *domain.CustomLoginText) {
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewBackButtonText {
+		result.ExternalRegistrationUserOverview.BackButtonText = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewDescription {
+		result.ExternalRegistrationUserOverview.Description = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewEmailLabel {
+		result.ExternalRegistrationUserOverview.EmailLabel = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewFirstnameLabel {
+		result.ExternalRegistrationUserOverview.FirstnameLabel = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewLanguageLabel {
+		result.ExternalRegistrationUserOverview.LanguageLabel = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewLastnameLabel {
+		result.ExternalRegistrationUserOverview.LastnameLabel = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewNextButtonText {
+		result.ExternalRegistrationUserOverview.NextButtonText = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewNicknameLabel {
+		result.ExternalRegistrationUserOverview.NicknameLabel = text.Text
+	}
+	if text.Key == domain.LoginKeyExternalRegistrationUserOverviewNicknameLabel {
+		result.ExternalRegistrationUserOverview.NicknameLabel = text.Text
+	}
```

Comment:
duplicate (just above)

Reply:
removed and check if any other text is missing